error[E0432]: unresolved import `layer_wasi::ethereum::WasiProvider`

following the README on a fresh machine, i get the following error at the `make build` step:

```
$ make build

[⠊] Compiling...
No files changed, compilation skipped
Generating bindings for 39 contracts
Bindings have been generated to ./crates/bindings
warning: profiles for the non root package will be ignored, specify profiles at the workspace root:
package:   /Users/noah/Developer/layer/wavs-foundry-template/components/eth-trigger-weather/Cargo.toml
workspace: /Users/noah/Developer/layer/wavs-foundry-template/Cargo.toml
   Compiling bindings v0.1.0 (/Users/noah/Developer/layer/wavs-foundry-template/crates/bindings)
   Compiling eth-trigger-weather v0.1.0 (/Users/noah/Developer/layer/wavs-foundry-template/components/eth-trigger-weather)
error[E0432]: unresolved import `layer_wasi::ethereum::WasiProvider`
 --> components/eth-trigger-weather/src/trigger.rs:8:5
  |
8 |     ethereum::WasiProvider,
  |     ^^^^^^^^^^^^^^^^^^^^^^ no `WasiProvider` in `ethereum`

warning: unused import: `ISimpleSubmit::DataWithId`
  --> components/eth-trigger-weather/src/trigger.rs:45:13
   |
45 |     pub use ISimpleSubmit::DataWithId;
   |             ^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: `#[warn(unused_imports)]` on by default

For more information about this error, try `rustc --explain E0432`.
warning: `eth-trigger-weather` (lib) generated 1 warning
error: could not compile `eth-trigger-weather` (lib) due to 1 previous error; 1 warning emitted
warning: build failed, waiting for other jobs to finish...
make: *** [build] Error 101
```

my rust version is `1.84.0` which is the current stable:
```
$ rustc --version
rustc 1.84.0 (9fc6b4312 2025-01-07)
```

--- a/Makefile
+++ b/Makefile
@@ -13,8 +13,8 @@ bindings:
 		--alloy --alloy-version v0.9.2
 
 ## build: building the project
-build: bindings
-	@$(CARGO) build
+build: bindings wasi-build
+	@$(CARGO) build --target-dir ./target --manifest-path ./app/Cargo.toml
 
 ## wasi-build: building the WAVS wasi component(s)
 wasi-build:

--- a/README.md
+++ b/README.md
@@ -17,7 +17,9 @@ A comprehensive template for developing WAVS (WebAssembly AVS) applications usin
 Create a new project using this template:
 
 ```bash
-forge init --template https://github.com/Lay3rLabs/wavs-foundry-template my-wavs
+# If you don't have forge: `curl -L https://foundry.paradigm.xyz | bash`
+
+forge init --template Lay3rLabs/wavs-foundry-template my-wavs
 ```
 
 ## Testing

